lib/setupmanagers/qemuhck/storage_manager.rb

```diff
@@ -83,18 +84,32 @@ def boot_device_command(device_name, run_opts, qemu_replacement_map)
 
         options = {
           '@image_format@' => IMAGE_FORMAT,
-          '@image_path@' => image_path
+          '@image_path@' => image_path,
+          '@bootindex@' => ',bootindex=1'
         }
 
         [device_command_info('boot', device_name, options, qemu_replacement_map), image_path]
       end
 
+      def iso_commands(run_opts, _qemu_replacement_map)
+        boot_index = 1
+
+        run_opts[:attach_iso_list]&.map do |iso|
+          iso_path = Pathname.new(@iso_path).join(iso)
+          boot_index += 1
+
+          "-device ide-cd,drive=iso_drive_#{boot_index},bus=ide.#{boot_index},bootindex=#{boot_index} " \
+            "-drive file=#{iso_path},if=none,media=cdrom,readonly=on,id=iso_drive_#{boot_index}"
+        end
+      end
+
```

Comment:
Why would we want `bootindex` for ISOs?

Reply:
It's better to specify `bootindex` only for the install disk though I don't require to make a change for that.

Reply:
Yes, this is a better solution,  but storage_manager and qemuhck do not know what image is really bootable.

Reply:
A simple approach is to introduce a new option like `attach_installer_iso`.

Reply:
Will move to separate task